chroma_core/models/host.py

```diff
@@ -1785,11 +1785,33 @@ class NoNidsPresent(Exception):
     pass
 
 
+class CreateSnapshotStep(Step):
+    def run(self, kwargs):
+        self.invoke_rust_agent_expect_result(
+            kwargs["host"],
+            "snapshot_create",
+            {"fsname": kwargs["fsname"], "name": kwargs["name"], "comment": kwargs["comment"]},
```

Comment:
will it fail if comment is not defined?

Reply:
I guess it will, but our arguments structure doesn't support not having a comment.

Reply:
It's Option though https://github.com/whamcloud/integrated-manager-for-lustre/commit/4c0b714950f5de892ae7beb4da9dc7c34df6a9a0#diff-da0597742d3aa1093abb8b957bb71a81R21

Reply:
You could check if `kwargs`  has a "comment" key and add it to the dictionary if so.

Reply:
Ah, must've mistaken for something else.

Fixed.